spec/requests/toons_spec.rb

```diff
@@ -37,5 +37,25 @@
       toon = Toon.first
       expect(toon.name).to eq('Patrick')
     end
+
+    it 'will not create a toon without a name' do
+      toon_params = {
+        toon: {
+          age: 35,
+          enjoys_doing: 'sleep',
+          image: 'www.com'
+        }
+      }
```

Comment:
Since the test is to check for the name error message, make sure that the other keys are not generating errors too. Recommend modifying the value for `enjoys_doing` key so it does not trigger the minimum length validation.